.ci/ogc/Dockerfile

```
@@ -1,7 +1,6 @@
 FROM ubuntu:latest
-MAINTAINER Paul Blottiere <blottiere.paul@gmail.com>
 RUN  export DEBIAN_FRONTEND=noninteractive
-ENV  DEBIAN_FRONTEND noninteractive
+ENV  DEBIAN_FRONTEND=noninteractive
```

```suggestion
ARG DEBIAN_FRONTEND=noninteractive
```

Minor suggestion
See https://serverfault.com/questions/618994/when-building-from-dockerfile-debian-ubuntu-package-install-debconf-noninteract